src/main/java/com/chekanova/imagetool/validation/HexValidationUtil.java

```diff
@@ -0,0 +1,26 @@
+package com.chekanova.imagetool.validation;
+
+import lombok.experimental.UtilityClass;
+import org.springframework.web.servlet.mvc.support.RedirectAttributes;
+
+import java.util.Locale;
+import java.util.ResourceBundle;
+import java.util.regex.Pattern;
+
+@UtilityClass
+public class HexValidationUtil {
+
```

Comment:
Please rename FileValidationUtil to ValidationUtil and move all validation to that class

Reply:
done